release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -363,6 +363,20 @@ Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable,
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-10-cluster-cloud-controller-manager-operator"]
+==== Cloud controller managers for additional cloud providers
+
+The Kubernetes controller manager that manages cloud provider deployments does not include support for Alibaba Cloud or IBM Cloud as providers. Because using cloud controller managers is the preferred method for interacting with underlying cloud platforms, there is no plan to add this support. As a result, the Alibaba Cloud and IBM Cloud implementations in {product-title} use cloud controller managers.
+
+In addition, this release supports using cloud controller managers for Google Cloud Platform (GCP) and VMware vSphere as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. Any new cloud platform support that is added to {product-title} will also use cloud controller managers.
```

Comment:
I would soften the wording on this second sentence, since this is a future prediction. Maybe work in "plan" in there?